From 7b1099c63a7a2f9ffdf4f1840f4ad52e8daa7726 Mon Sep 17 00:00:00 2001
From: Samuel Dionne-Riel <samuel@dionne-riel.com>
Date: Fri, 31 Dec 2021 18:06:38 -0500
Subject: [PATCH 2/3] =?UTF-8?q?[WIP]=20redmi=20=E2=86=92=20xiaomi?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 BoardConfig.mk    | 2 +-
 potato_begonia.mk | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/BoardConfig.mk b/BoardConfig.mk
index ee6e4c9..3a6bb78 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-DEVICE_PATH := device/redmi/begonia
+DEVICE_PATH := device/xiaomi/begonia
 
 # Architecture
 TARGET_ARCH := arm64
diff --git a/potato_begonia.mk b/potato_begonia.mk
index 499e8c8..fb488ad 100644
--- a/potato_begonia.mk
+++ b/potato_begonia.mk
@@ -25,7 +25,7 @@ $(call inherit-product, $(SRC_TARGET_DIR)/product/updatable_apex.mk)
 $(call inherit-product, $(SRC_TARGET_DIR)/product/gsi_keys.mk)
 
 # Inherit from begonia device
-$(call inherit-product, device/redmi/begonia/device.mk)
+$(call inherit-product, device/xiaomi/begonia/device.mk)
 
 # Inherit from our custom product configuration
 $(call inherit-product, vendor/potato/config/common_full_phone.mk)
-- 
2.34.0

