From 71920a6f19e400542d8d9d65d07f479409300e3b Mon Sep 17 00:00:00 2001
From: Samuel Dionne-Riel <samuel@dionne-riel.com>
Date: Thu, 30 Dec 2021 23:06:47 -0500
Subject: [PATCH 24/24] [my] aura: Remove Snap app

---
 device.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/device.mk b/device.mk
index 395bd8d..bf885ac 100644
--- a/device.mk
+++ b/device.mk
@@ -169,7 +169,6 @@ PRODUCT_PACKAGES_DEBUG += \
 PRODUCT_PACKAGES += \
     android.hardware.camera.provider@2.4-impl:32 \
     android.hardware.camera.provider@2.4-service \
-    Snap \
     vendor.qti.hardware.camera.device@1.0.vendor \
     camera.device@3.2-impl \
     camera.sdm845 \
-- 
2.34.0

